### ISSUE TITLE
[cinder] include support for cinder-backup service?

### ISSUE THREAD

**Issue by mancdaz** [rcbops/chef-cookbooks#734]
Do we want to add support for running cinder-backup?  This allows backup of cinder volumes to some other back end store (eg swift, rbd)

**Comment by odyssey4me** [comment]
Absolutely. In fact, requests for some sort of scheduled backups/snapshots have come through reasonably often. Gridcentric has a base of code for this, although I have no idea how portable it is.
